src/sentry/tasks/derive_code_mappings.py

```diff
@@ -68,20 +69,23 @@ def derive_code_mappings(
         report_project_codemappings(code_mappings, stacktrace_paths, project)
         return
 
-    set_project_codemappings(code_mappings, organization_integration, project)
+    return set_project_codemappings(code_mappings, organization_integration, project)
 
 
 def identify_stacktrace_paths(data: NodeData) -> List[str]:
     """
     Get the stacktrace_paths from the event data.
     """
-    if data["platform"] != "python":
+    if data["platform"] not in SUPPORTED_LANGUAGES:
         return []
+
     stacktraces = get_stacktrace(data)
     stacktrace_paths = set()
     for stacktrace in stacktraces:
         try:
-            paths = {frame["filename"] for frame in stacktrace["frames"]}
+            paths = {
+                frame["filename"] for frame in stacktrace["frames"] if frame.get("in_app") is True
```

Comment:
This allows ignoring frames that the SDK has marked as a 3rd party module.
At the moment, the Python SDK marks all frames as in_app, thus, it was not necessary.